### PR TITLE
Add Symfony 5 support

### LINKED ISSUE #11: Not compatible with Symfony 5
```
roave/behat-psr11extension 1.1.0 requires symfony/dependency-injection ^2.7 || ^3.0 || ^4.0 -> no matching package found
```

### PR DESCRIPTION
**Pull request by snapshotpl**
Fixes #11

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "php": "^7.1",
         "behat/behat": "^3.3",
         "psr/container": "^1.0",
-        "symfony/dependency-injection": "^2.7 || ^3.0 || ^4.0"
+        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0"
```
Comment (Ocramius) [inline composer.json]: Let's bump to `^5` only for newer releases, since older versions will still work with previous symfony stacks

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (snapshotpl) [reply composer.json]: Give me a sec

Comment (Ocramius) [reply composer.json]: Nono, I mean supporting **only** `^5` 😅

**Comment by snapshotpl** [comment]
@Ocramius all green. Merge and release? :-)

**Comment by Ocramius** [comment]
Wait, behat with symfony 5 support was already released?!

**Comment by Ocramius** [comment]
Oh yeah, 3.6.0 is out 🤘

**Comment by snapshotpl** [comment]
https://github.com/Behat/Behat/releases/tag/3.6.0

**Pull request merged**